org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -217,7 +217,7 @@ Mutation:
     ('mutation')
     ('(' (triggers+=TriggerRef (',' triggers+=TriggerRef)*)? ')')?
     (sources+=VarRef (',' sources+=VarRef)*)?
-    ('->' effects+=[VarRef] (',' effects+=[VarRef])*)?
+    ('->' effects+=VarRef (',' effects+=VarRef)*)?
```

Comment:
Just out of curiosity, what is the implication of this change?

Reply:
Marten checked that it was just a mistake.
The VarRef already list variable.